docs/receiving/receiving-with-ingest.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Receiving Webhooks with Ingest
+---
+
+import CodeTabs from '@theme/CodeTabs';
+import TabItem from '@theme/TabItem';
+
+[Svix Ingest] grants you scalability, security best practices, and
+reliability, all with minimal configuration.
```

Comment:
```suggestion
reliability for receiving webhooks, all with minimal configuration.
```

I know it's redundant, but the sentence feels generic/weird without saying 'for what'

Reply:
Yeah I agree -- "grants you" is also a bit awkward. Maybe the whole thing should be re-worded.

Reply:
I redid the opening. It's different now, and hopefully less awkward.